config/routes.rb

```diff
@@ -120,6 +120,8 @@ def scoped_navigation_routes(context, navigation, as_redirects: false)
       resources :clients do
         get "/organization", to: "clients/organizations#edit", on: :member, as: :edit_organization
         patch "/organization", to: "clients/organizations#update", on: :member, as: :organization
+        get "/bai", to: "clients/bank_accounts#show", on: :member, as: :show_bank_account
```

Comment:
Non-blocker: wondering if "bai" acronym is enough or if it would be better to just spell out `bank_account_info`.